build.sbt

```diff
@@ -1,15 +1,42 @@
-val specs2Version = "4.9.2"
-val http4sVersion = "0.21.3"
+ThisBuild / scalaVersion := "2.13.1"
+
+ThisBuild / crossScalaVersions += "2.12.11"
+
+ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.8", "adopt@1.11")
+ThisBuild / githubWorkflowPublishTargetBranches := Seq() // Disable "publish" job generation
```

Comment:
The publish workflow generated by sbt-github-actions by default runs on every change to master. Going by apparent convention in specs2 to not publish SNAPSHOTs and leave the version in the build file set to the last released version this is probably not what you want, so I've disabled it.